### ISSUE TITLE
Unable to dismiss notification to switch to new version. Unable to use new version without elementor account.

### ISSUE THREAD

**Issue by rainbowforgedev** [elementor/one-click-accessibility#231]
I do not want to create an elementor account. I just want to use the accessibility plugin for accessibility. I can't even dismiss the notification demanding that I switch to the new version. Please make available the code for the original oneclick plugin without all the elementor advertising.

**Comment by ip00** [comment]
I would also like to use it without an elementor account and avoid having a cookie: __cf_bm | .elementor.com (This cookie, set by Cloudflare, is used to support Cloudflare Bot Management.).
I don’t see why an account or site linking should be necessary for this kind of functionality. Plus, since it’s advertised as ‘forever free’ by your staff on WordPress.org, this requirement feels a bit unnecessary and raises privacy concerns.

**Comment by bainternet** [comment]
Simply Head to plugins and you will see a link under the Ally plugin "Revet to legacy".

**Comment by rainbowforgedev** [comment]
That's an inadequate response. When that is done, there is a banner at the top of the customizer that cannot be dismissed suggesting switching to the updated version. This is an accessibility plugin for chrissake, can the good folks at Elementor stop being greedy for _once_?
